src/Tools/BuildBoss/BuildBoss.csproj

```diff
@@ -11,6 +11,7 @@
   <ItemGroup>
     <Reference Include="System" />
     <Reference Include="System.Core" />
+    <Reference Include="System.IO.Compression" />
```

Comment:
😕 Was this change required?

Reply:
Yes, it seems that ZipArchive was moved from one DLL to another and this generates a new compiler warning without the reference.

Reply:
This might indicate a packaging issue with whatever had the demand -- that could have been listed in the NuGet package itself. Not a big deal though.